spark/src/main/scala/org/apache/spark/sql/delta/DeltaParquetFileFormat.scala

```diff
@@ -417,6 +416,12 @@ object DeltaParquetFileFormat {
   val ROW_INDEX_COLUMN_NAME = "__delta_internal_row_index"
   val ROW_INDEX_STRUCT_FIELD = StructField(ROW_INDEX_COLUMN_NAME, LongType)
 
+  /** The name of the metadata column that contains the encoded row index filter identifier. */
```

Comment:
Update this comment and the one below, this isn't the name of a metadata column anymore, just the key for the row index filter id / type in the file's `otherConstantMetadataColumnValues` map.

Reply:
@johanl-db My bad yeah